content/general/concepts/ada-lovelace/ada-lovelace.md

```diff
@@ -6,14 +6,17 @@ Subjects:
   - 'Code Foundations'
```

Comment:
While we're in here, I would fix the punctuation in the description on line 3. "Charles Babbages" should be "Charles Babbage's".

Reply:
Gotcha. I think I might have left that out to prevent a check fail but I think I can probably remove his name entirely from the Description 😄